src/library/components/Tabs.tsx

```diff
@@ -1,4 +1,4 @@
-// TODO: figure out if we want to add aria-label or aria-labelledby on the tablist
+// TODO: ADS-384 figure out if we want to add aria-label or aria-labelledby on the tablist
```

Comment:
Can we update this comment to:
TODO: ADS-384 conditionally add aria-labelledby or aria-label on the tablist